TarSCM/scm/base.py

```diff
@@ -249,4 +276,5 @@ def unlock_cache(self):
             self.lock_file = None
 
     def finalize(self):
+        self.cleanup()
         pass
```

Comment:
The "pass" statement should be obsolete then.

Reply:
You're right, this is redundant.